web-components/src/mixins/FocusTrapMixin.test.ts

```diff
@@ -424,17 +424,21 @@ describe("FocusTrap Mixin", () => {
     const focusTrap = elWithCB.shadowRoot!.querySelector<FocusTrap>("focus-trap");
     const combobox = focusTrap!.querySelector("md-combobox");
 
+    const input = combobox!.shadowRoot?.querySelector("input");
+
     await elementUpdated(elWithCB);
-    console.log("aaaaaaaaa", focusTrap?.innerHTML);
+    console.log("focusTrap innerHTML:", focusTrap?.innerHTML);
```

Comment:
Do we need console logs in test cases? We should remove it.

Reply:
it was already there actually, so i just made it meaningful.. let me remove it